fivetran_provider_async/hooks.py

```diff
@@ -102,6 +105,9 @@ async def get_sync_status_async(self, connector_id, previous_completed_at):
         :param previous_completed_at: The last time the connector ran, collected on Sensor
             initialization.
         :type previous_completed_at: pendulum.datetime.DateTime
+        :param reschedule_time: Optional, if connector is in reset state
+            number of seconds to wait before restarting, else Fivetran suggestion used
```

Comment:
I am confused with the "Fivetran suggestion used" statement. What is the Fivetran suggestion? Do we mean Fivetran suggests some wait time or by default some wait time is configured in the connector? If that is the case, should we not set this to `None` instead of setting it to `0`?

Reply:
I have rephrased it: https://github.com/astronomer/airflow-provider-fivetran-async/pull/25/commits/202548d3de470c8ba2d3086c64cbb1082a27f6b7